art/attacks/fast_gradient.py

```diff
@@ -88,8 +88,9 @@ def _minimal_perturbation(self, x, y):
 
         :param x: An array with the original inputs
         :type x: `np.ndarray`
-        :param y:
-        :type y:
+        :param y: Target values (class labels) one-hot-encoded of shape (nb_samples, nb_classes) or indices of shape
```

Comment:
Internally, ART is treating target values as on-hot-encoded which is ensured by `check_and_transform_label_format` in line 141. Could you please remove `or indices of shape (nb_samples,)`?
```suggestion
        :param y: Target values (class labels) one-hot-encoded of shape (nb_samples, nb_classes)
```